src/Isolation/Masa.Contrib.Isolation.MultiTenancy/README.md

```diff
@@ -0,0 +1,91 @@
+[中](README.zh-CN.md) | EN
+
+## Masa.Contrib.Isolation.MultiTenancy
```

Comment:
MultiTenant? MultiTenancy?

Reply:
Uniformly modified to MultiTenant, processed